src/components/atoms/TooltipArrow.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core';
+
+export enum ArrowDirection {
+  TOP,
```

Comment:
`TooltipOffset` and `ArrowDirection` enums have the same values.
You can use the same enum, and externalize it to util file `utils\enum.utils.ts`.
you can name the enum `direction` or [ClockPosition](https://en.wikipedia.org/wiki/Clock_position)